jplag.frontend.scala/src/main/scala/de/jplag/scala/ScalaToken.scala

```diff
@@ -0,0 +1,36 @@
+package de.jplag.scala
+
+import de.jplag.Token
+
+import scala.meta.inputs.Position
+
+class ScalaToken(tType: Int,
+                 file: java.lang.String,
+                 var line: Int,
+                 var column: Int,
+                 var length: Int)
+  extends Token(tType, file, line, column, length) {
+
+    def this(tType: ScalaTokenConstants.Value, file: String, pos: Position, fromEnd: Boolean) {
```

Comment:
I don't know about the rules in frontends, but abbreviations like `pos` for `position` are generally viewed as bad practice (*write once, read many*)

Reply:
Absolutely! Coming right up

Reply:
Absolutely! I gave the identifiers a good look again, hopefully I got them all.